app/code/community/Aligent/NewRelicFix/etc/system.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * system.xml
+ *
+ * @category  Aligent
+ * @package   Aligent_Storelocator
+ * @author    Brent Honeybone <brent@aligent.com.au>
+ * @copyright 2014 Aligent Consulting.
+ * @license   All Rights Reserved
+ * @link      http://www.aligent.com.au/
+ */
+-->
+<config>
+    <sections>
+        <aligent_newrelicfix translate="label">
+            <label>NewRelic</label>
+            <tab>aligent</tab>
+            <frontend_type>text</frontend_type>
+            <sort_order>2000</sort_order>
+            <show_in_default>1</show_in_default>
+            <show_in_website>1</show_in_website>
+            <show_in_store>1</show_in_store>
+            <groups>
+                <trigger_error translate="label">
+                    <label>Trigger Error</label>
+                    <sort_order>20</sort_order>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>1</show_in_website>
+                    <show_in_store>1</show_in_store>
+                    <fields>
+                        <disable translate="label">
+                            <label>Disable Trigger Error</label>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <sort_order>10</sort_order>
+                            <show_in_default>1</show_in_default>
```

Comment:
@zainengineer It's not clear to me why this switch exists.  If the intention is to have an empty controller which be edited to call the helper functions to create errors which can be logged into  NR for testing, then why have a switch to disable the test code?  Wouldn't having a switch to disable it mean that if an error isn't logged I now have two possible concerns, 1) That NR logging isn't working, and 2) my test code is disabled because of a system config value (or stale cached copy thereof)?  Could you perhaps elaborate a little further on your reasoning for wanting this switch?